### DISCUSSION TITLE
Using custom directives in Quasar QTable

### DISCUSSION THREAD

**Discussion by pgmo** [quasarframework/quasar#10342] [General - Components / Directives / etc]
Hi everyone,
quite new in the Quasar world. I'm trying to use one of my custom directives in a **QTable** in order to apply markdown to a text field through **markedjs**. The custom directive is called `marked` and looks like this:
```
.directive('marked', {

        updated(el) {
            el.innerHTML = marked(el.innerHTML, {
            ... some_options ...
            }
```
To apply the `marked` directive to the table cell I use a code like this:
```
<template v-slot:body-cell-ActivityDescription="props">
      <q-td auto-width style="white-space:nowrap">
        <div v-marked>
          {{ props.row.ActivityDescription }}
        </div>
      </q-td>
</template>
```
What I see is that the first table load runs as expected. However when I add an item to the table and refresh its content, rows list is not correct - some rows get duplicated, depending on where the new row should be inserted, the new description does not appear - total number of rows is correct. If I press F5 to refresh, the list appears ok again.
I tried any other hook but with no success.

Any suggestion?

Thanks in advance, regards

**Comment by ontwikkelfabriek** [comment]
@pgmo 

In years of using Quasar I never had to resolve to creating a custom directive ( directly manipulating the DOM == bad practice in Vue). Instead I would use a filter or a method call for this.

**Reply by pgmo** [reply]
Ok, many thanks for your suggestion
